images/win/scripts/Tests/PowerShellModules.Tests.ps1

```diff
@@ -0,0 +1,23 @@
+Describe "PowerShell Modules" {
+    $modules = (Get-ToolsetContent).powershellModules
+    $withoutVersionsModules = $modules | Where-Object {-not $_.versions} | ForEach-Object {
+        @{moduleName = $_.name}
+    }
+
+    $withVersionsModules = $modules | Where-Object {$_.versions} | ForEach-Object {
+        $moduleName = $_.name
+        $_.versions | ForEach-Object {
+            @{moduleName = $moduleName; expectedVersion = $_}
+        }
+    }
+
+    It "<moduleName> is installed" -TestCases $withoutVersionsModules {
+        Get-Module -Name $moduleName -ListAvailable | Should -BeTrue
+    }
+
+    if ($withVersionsModules) {
+        It "<moduleName> with <expectedVersion> is installed" -TestCases $withVersionsModules {
+            (Get-Module -Name $moduleName -ListAvailable).Version -eq $expectedVersion | Should -BeTrue

```

Comment:
I could be wrong, but it won't work if we specify more than 1 version?
In this case, `(Get-Module -Name $moduleName -ListAvailable).Version` returns array of versions.
And `-eq` array with 1 version will fail.

Reply:
It works fine, but maybe the contains operator is more obvious.
```
PS > @(1,2,3,4) -eq 1
1
PS > @(1) -eq 1
1
PS > @(1) -contains 1
True
PS > @(1,2,3,4) -contains 1
True
PS > (1,2,3,4) -eq 1 | Should -BeTrue
PS > (1) -eq 1 | Should -BeTrue
```